lib/fog/compute/serverlove.rb

```diff
@@ -1,11 +1,11 @@
 module Fog
   module Compute
     class Serverlove < Fog::Service
-      autoload :Image, 'fog/compute/serverlove/models/image'
-      autoload :Images, 'fog/compute/serverlove/models/images'
-      autoload :Server, 'fog/compute/serverlove/models/server'
-      autoload :Servers, 'fog/compute/serverlove/models/servers'
-      autoload :PasswordGenerator, 'fog/compute/serverlove/password_generator'
+      autoload :Image, File.expand_path('../serverlove/models/image', __FILE__)
+      autoload :Images, File.expand_path('../serverlove/models/images', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.